packages/cli/lang/en.lyaml

```diff
@@ -1046,6 +1046,14 @@ en:
       betaWarning:
         header: "{{#yellow}}***************************** WARNING ****************************{{/yellow}}"
         footer: "{{#yellow}}******************************************************************{{/yellow}}"
+      infoTag: "{{#bold}}[INFO]{{/bold}}"
+      deprecatedTag: "{{#bold}}[DEPRECATED]{{/bold}}"
+      errorTag: "{{#bold}}[ERROR]{{/bold}}"
+      deprecatedMessage: "The `{{#orange}}{{ command }}{{/orange}}` command is deprecated and will be disabled soon. {{ url }}"
+      deprecatedDescription: "{{ message }}. The `{{#orange}}{{ command }}{{/orange}}` command is deprecated and will be disabled soon. {{ url }}"
+      deprecatedUrlText: 'Learn more.'
+      disabledMessage: "The `{{#orange}}{{ command }}{{/orange}}` command is disabled. Run `{{#orange}}npm i -g @hubspot/cli@latest{{/orange}}` to update to the latest HubSpot CLI version. {{ url }}"
+      disabledUrlText: "See all HubSpot CLI commands here."
```

Comment:
I like keeping all of this here 👌 

Just a note though, the error and info tags should already be natively supported by the logger [here](https://github.com/HubSpot/hubspot-local-dev-lib/blob/main/lib/logger.ts#L34). You can use them via `logger.error()` and `logger.info()`. This is a pattern we probably want to keep because there are some nuances to the behavior of logging via `.log()` vs. `.error()`. Logging as an error will log out via process.stderr, and will respect the console log level visibility controls